cmd/cinder-csi-plugin/main.go

```diff
@@ -80,7 +80,7 @@ func main() {
 	cmd.PersistentFlags().StringVar(&endpoint, "endpoint", "", "CSI endpoint")
 	cmd.MarkPersistentFlagRequired("endpoint")
 
-	cmd.PersistentFlags().StringVar(&cloudconfig, "cloud-config", "", "CSI driver cloud config")
+	cmd.PersistentFlags().StringSliceVar(&cloudconfig, "cloud-config", nil, "CSI driver cloud config")
```

Comment:
based on this , I think we might need upgrade CSI version defined at 
https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/development.md

Reply:
Yes. Now that you mention it this is also critical to the operator, as the change is backwards compatible but not forwards compatible. I've bumped the version. As it's the first version which no longer follows the CSI Spec version I've also reformatted the comment to include a version history. Having previously worked with nova I have frequently found this to be useful in practise: https://opendev.org/openstack/nova/src/commit/fa59cde51ffd761ac552b93af6c67e2432288548/nova/compute/rpcapi.py#L76

I also updated the help text of the cloudconfig param to note that it can be given multiple times.